components/shared/embedder/lib.rs

```diff
@@ -585,3 +589,57 @@ pub enum LoadStatus {
     /// See <https://developer.mozilla.org/en-US/docs/Web/API/Document/readyState>
     Complete,
 }
+
+/// <https://notifications.spec.whatwg.org/#notifications>
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct Notification {
```

Comment:
The documentation for this should be more complete since this exposed to the embedding API.